garrysmod/lua/includes/modules/scripted_ents.lua

```diff
@@ -149,14 +149,16 @@ function Get( name )
 	
 	-- Derive from base class
 	if ( name != SEntList[ name ].Base ) then

```

Comment:
If you're suggesting the if statement here isn't neccesary, I assume it's so that we don't get any infinite loops on the base entities (base_anim, base_point and base_brush).